libcontainerd/client_windows.go

```diff
@@ -294,13 +294,13 @@ func (clnt *client) Signal(containerID string, sig int) error {
 	if syscall.Signal(sig) == syscall.SIGKILL {
 		// Terminate the compute system
 		if err := cont.hcsContainer.Terminate(); err != nil {
-			if err != hcsshim.ErrVmcomputeOperationPending {
+			if !hcsshim.IsPending(err) {
 				logrus.Errorf("libcontainerd: failed to terminate %s - %q", containerID, err)
 			}
 		}
 	} else {
 		// Terminate Process
-		if err := cont.hcsProcess.Kill(); err != nil {
+		if err := cont.hcsProcess.Kill(); err != nil && !hcsshim.IsAlreadyStopped(err) {
```

Comment:
Would be nice to make `IsAlreadyStopped()` check for `nil` (similar to https://golang.org/src/os/error_unix.go?h=isNotExist#L25), then checking for nil can be skipped here, and simply

```golang
if err := cont.hcsProcess.Kill(); !hcsshim.IsAlreadyStopped(err) {
```